express/scripts/scripts.js

```diff
@@ -1480,11 +1480,11 @@ async function loadLazy() {
 function loadIMS() {
   window.adobeid = {
     client_id: 'MarvelWeb3',
-    scope: 'AdobeID,openid',
+    scope: 'AdobeID,openid,creative_sdk,gnav,sao.spark,additional_info.projectedProductContext,tk_platform,tk_platform_refresh_user,tk_platform_sync,creative_cloud,ab.manage,sao.typekit,mps,read_organizations,stk.a.k12_access.cru,stk.a.limited_license.cru,DCAPI,sao.ACOM_CLOUD_STORAGE',
     locale: getLocale(window.location),
     environment: 'prod',
   };
-  loadScript('https://auth.services.adobe.com/imslib/imslib.min.js');
+  loadScript('auth.services.adobe.com/imslib/imslib.min.js');
```

Comment:
@KushalPandya-JS without at least a `//` scheme, this request ends up being a 404:
`https://quick-action-update--express-website--kushalpandya-js.hlx.page/express/auth.services.adobe.com/imslib/imslib.min.js`
Btw, it appears that `imslib` gets loaded in `gnav.js` already, so I think we can remove this line completely...

Reply:
Thanks @rofe for the suggestion

Reply:
@KushalPandya-JS as predicted, the `imslib` now gets loaded twice in the test page provided above. I think line 1487 can be removed from `scripts.js`.

Also, the page now fails to load. You probably need to merge `main` into your branch again.

Reply:
Okay, I've merged main